spec/users_helpers_spec.rb

```diff
@@ -0,0 +1 @@
+#What goes in here? 
```

Comment:
The fact this has no generated code either makes me wonder if you accidentally made a nearly-empty copy `spec/helpers/users_helper_spec.rb`